packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -164,9 +164,15 @@ const runMigrationCommand: CliCommandDefinition<CreateFlags> = {
 
     const projectConfig = apiClient({
       requireUser: true,
-      requireProject: true,
+      requireProject: false,
```

Comment:
I left this in as `false` instead of removing it so that it would indicate that `false` is intended here.